### ISSUE TITLE
[Accessibility][Bypass Blocks - Property Pane]: Ensure frame must have tittle attribute.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2143]
**User Experience** 
Frame title should be define properly so that user can get the information of control.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
5. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
6. Navigate to property pane appeared at right side of the screen and select 'Bot Asks'.
7. Run Accessibility insight and observe the issue.

**Actual:** 
Adaptive property frame doesn't contain any tittle attribute. 

**Expected:**
Ensures <iframe> and <frame> elements contain a non-empty title attribute.

**Note:**
This accessibility issue was found using Accessibility Insights for Web 2.14.1 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Element path**
.css-vaviek

**Snippet**
`<iframe name="FormEditor" src="/extensionContainer.html" class="css-vaviek"></iframe>`

**How to fix**
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute or the title attribute is empty
      Element's default semantics were not overridden with role="presentation"
      Element's default semantics were not overridden with role="none"

**Note**
Same issue observed throughout the website for every property pane as well as the flow design.

**MAS Impacted:** [MAS2.4.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={20bce2aa-9952-4a10-af79-3376bcc8766d})

**Attachment:**
![frame tittle](https://user-images.githubusercontent.com/60913131/75655135-3c495280-5c87-11ea-8e15-1ff099e25201.png)
![Bypass block](https://user-images.githubusercontent.com/60913131/75655434-ddd0a400-5c87-11ea-92ad-e5270a305279.png)
![Bypass block property pane](https://user-images.githubusercontent.com/60913131/75655603-3acc5a00-5c88-11ea-85ac-878214dd4cdb.png)

**Comment by ashish315** [comment]
@cwhitten As validated, some issue's are fixed but some are still repro. If you want, we can log separate bug for the same and close this bug.
Issue still repro-
1. Ensures each page has at least one mechanism for a user to bypass navigation and jump straight to the content.

**Comment by corinagum** [comment]
@ashish315 for clarity, I think it would be great if you could file a new issue. Once you do, please add the link here so we don't miss anything. 

I'll close this as resolved once the new issue is filed. Thank you!

**Comment by ashish315** [comment]
@corinagum Please find the bug Id of new bug filled for the above issue.
[Accessibility][Bypass Blocks - Design Flow]: Ensures each page has at least one mechanism for a user to bypass navigation and jump straight to the content at property pane. #[2538](https://github.com/microsoft/BotFramework-Composer/issues/2538)

**Comment by cwhitten** [comment]
Thank you @ashish315. Please close this one.

**Comment by ashish315** [comment]
Closing the bug as per developer comment.
